CHANGELOG.md

```diff
@@ -74,20 +53,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Add support for the domain user attribute [(#127)](https://github.com/ManageIQ/manageiq-appliance/pull/127)
 - ExecStart requires an absolute executable path [(#151)](https://github.com/ManageIQ/manageiq-appliance/pull/151)
 
-## Unreleased as of Sprint 77 ending 2018-01-15
-
-### Added
-- Adding alias apcli for appliance_conosle_cli [(#175)](https://github.com/ManageIQ/manageiq-appliance/pull/175)
-
-### Fixed
-- Update manageiq-appliance_console to minimum v1.2.3 [(#168)](https://github.com/ManageIQ/manageiq-appliance/pull/168)
```

Comment:
This was the only entry for this PR in the changelog ... it looks like it was backported to Gaprindashvili.

Reply:
It's okay that we don't have that PR listed. There are certain PRs with labels that we don't publish in the changelogs.